cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/cloudstorage/Bundle.java

```diff
@@ -42,7 +42,7 @@ public class Bundle
 {
     private static final String MANIFEST_FILE_NAME = "manifest.json";
 
-    public final BigInteger startToken;
+    public final BigInteger firstToken;
```

Comment:
The start tokens in the ranges are usually excluded, since the ranges are open-closed unless called out specifically. The token here is really the first token in the range. Renaming to clarify.